lib/preloader/preloader.test.tsx

```diff
@@ -0,0 +1,53 @@
+import * as React from 'react';
+
+import Preloader from './preloader';
+
+import { mount } from 'enzyme';
+
+let wrapper;
+
+describe('<Preloader />', () => {
+  beforeEach(() => {
+    wrapper = mount(<Preloader />);
+  });
+
+  it('Handles props.active', () => {
+    expect(wrapper.getDOMNode().getAttribute('class')).toBe('oc-preloader');
```

Comment:
consider using constants... so that if you change the classes you dont need to go after the old string everywhere.

Reply:
Can I get you to give me a bit more context around this comment? Not sure that I understand what you're suggesting.

Reply:
as discussed, let's address this in a more generic way for the project after.